skyportal/utils/offset.py

```diff
@@ -797,6 +826,31 @@ def get_formatted_standards_list(
 
     if return_dataframe:
         return df
+    elif starlist_type == "P200-NGPS":
```

Comment:
a bit of a mess, but maybe hard to avoid.

Reply:
I think that the whole offset stuff in general is a mess, which causes something like this to happen. Everything was designed to duplicate / generalize but only for a specific ascii like format where all facilities essentially have the same columns. I'd be willing to redo the whole thing at some point, but for now the P200 folks just need this to be up and running for their observing runs